packages/gestalt/src/Icon/Icon.js

```diff
@@ -64,8 +65,6 @@ export default function Icon(props: Props) {
   } else if (dangerouslySetSvgPath) {
     /* eslint-disable-next-line no-underscore-dangle */
     path = dangerouslySetSvgPath.__path;
-  } else {
```

Comment:
React is fine leaving `path` undefined and the html markup actually renders better